specification/metadata/billinggenerator.md

```diff
@@ -0,0 +1,17 @@
+# Billing Generator
```

Comment:
Maybe Data Generator, or even simply FOCUS Generator?

FOCUS relates to Cost and Usage (not billing).

Reply:
+1
"Data Generator" will be more forward-looking as we expand FOCUS to other datasets.

Reply:
Updated